src/service_manager.rs

```diff
@@ -208,4 +208,28 @@ impl ServiceManager {
             Ok(Service::new(unsafe { ScHandle::new(service_handle) }))
         }
     }
+
+    /// Retrieves the service name of the specified service.
+    pub fn get_service_key_name(&self, display_name: impl AsRef<OsStr>) -> Result<OsString> {
+        let service_display_name =
+            WideCString::from_os_str(display_name).map_err(|_| Error::DisplayNameHasNulByte)?;
+        let mut size = 4u32 * 1024;
```

Comment:
Please leave a comment describing why this magic number was chosen. Like here https://github.com/mullvad/windows-service-rs/blob/24bd91a84529a315277b7096a52d67057a394edb/src/service.rs#L1466

Reply:
Done